server/utils/utils.ts

```diff
@@ -21,3 +23,32 @@ export const initialiseName = (fullName?: string): string | null => {
   const array = fullName.split(' ')
   return `${array[0][0]}. ${array.reverse()[0]}`
 }
+
+export function handleIntegrationErrors(status: number, message: string, integrationService: string): ErrorMessages {
```

Comment:
You're not using this function, and it looks like it's breach-specific - would it be best to remove for now and re-add it later if we need it?